app/src/main/res/layout/item_search_result.xml

```diff
@@ -25,9 +25,9 @@
             android:layout_marginEnd="16dp"
             android:ellipsize="end"
             android:maxLines="1"
-            android:text="@{String.valueOf(searchResult.name).concat(String.valueOf(',').concat(searchResult.subcountry)).concat(String.valueOf(',').concat(searchResult.country))}"
+            android:text="@{String.valueOf(searchResult.name).concat(String.valueOf(',')).concat(searchResult.country))}"
             android:textAppearance="?attr/textAppearanceBody1"
-            tools:text="Osogbo, Osun, Nigeria" />
+            tools:text="Osogbo, Nigeria" />
```

Comment:
So, we want to fully qualify the location the user is searching and that is why we are including a sub-country better still the state in the country.